ts/Series/PackedBubble/PackedBubbleSeries.ts

```diff
@@ -1473,7 +1473,6 @@ interface PackedBubbleSeries {
     forces: Array<string>;
     hasDraggableNodes: boolean;
     isCartesian: boolean;
-    noSharedTooltip: boolean;
```

Comment:
Q: Why removing props from the interface? By default `noSharedTooltip` is optional, but in this particular series it's always defined (and set to `true`). Probably for Booleans, it's not a big deal, but with other types, it might change (and require) more code.

Reply:
I removed `noSharedTooltip` from the `PackedBubbleSeries` interface because this property has been already declared in the `Tooltip.ts` file as a `SeriesLike` interface.

Should it (`noSharedTooltip` property) be declared in the interface for each series individually?

@bre1470 could you please help us decide which pattern is better?

Reply:
If it is possible, that a series with shared tooltips inherits from a series without shared tooltips, it is better to define the property more generic. In that case `SeriesLike` in `Tooltip.ts` should be enough.

Reply:
> Should it (noSharedTooltip property) be declared in the interface for each series individually?

Sorry, let me rephrase. Not for each series, but just the ones that overwrite the property. In other (coding) words:

```ts
interface SeriesLike {
  noSharedTooltip?: boolean
} 

class ColumnSeries extends SeriesLike {
}

class PieSeries extends ColumnSeries {
  noSharedTooltip: boolean // or even "true"
}

PieSeries.prototype.noSharedTooltip = true;

// Later in code:
// if (this.noSharedTooltip) { ... } <-- fine! 
```

As I said above, it's not a big deal with booleans, I just wanted to discuss this :) With e.g. objects it's a bit more troublesome:

```ts
interface customObjectInterface {
  x: number,
  y: number
}

interface SeriesLike {
  customObject?: customObjectInterface
} 

class ColumnSeries extends SeriesLike {
}

class PieSeries extends ColumnSeries {
  customObject: customObjectInterface
}

PieSeries.prototype.customObject = { x: 0, y: 0 };

// Later in code:
// `this` in an instance of PieSeries
if (this.customObject) {
   doSomething(
     this.customObject.x,
     this.customObject.y
   )
}
// Or
   doSomething(
     this.customObject?.x,
     this.customObject?.y
   )

// vs

doSomething(
  this.customObject.x,
  this.customObject.y
) // <-- `customObject` is defined
```